tests/karate/src/test/java/be/LAO/create.feature

```diff
@@ -7,47 +7,78 @@ Feature: Create a pop LAO
         # Meaning they share def variables, configurations ...
         # Especially JS functions defined in server.feature can be directly used here thanks to Karate shared scopes
     * call read('classpath:be/utils/server.feature')
+    * call read('classpath:be/mockFrontEnd.feature')
+    * call read('classpath:be/constants.feature')
     * def id = 1
     * string channel = "/root"
 
-  Scenario: Create Lao request with empty lao name should fail with an error response
-    Given string laoCreateData = read('classpath:data/lao/data/bad_lao_create_empty_name_data.json')
-    * string laoCreate = converter.publishМessageFromData(laoCreateData, id, channel)
-    And   def socket = karate.webSocket(wsURL,handle)
-    When  eval socket.send(laoCreate)
-    *  karate.log('Sent: '+ karate.pretty(laoCreate))
-    And  json err = socket.listen(timeout)
-    * karate.log('Received: '+ err )
-    Then match err contains deep {jsonrpc: '2.0', id: '#(id)', error: {code: -4, description: '#string'}}
+  Scenario: Create Lao request with empty lao name should fail with an error response 2
+    Given def badLaoReq =
+      """
+        {
+          "object": "lao",
+          "action": "create",
+          "id": '#(getLaoIdEmptyName)',
+          "name": "",
+          "creation": 1633098234,
+          "organizer": '#(getOrganizer)',
+          "witnesses": []
+        }
+      """
+    When frontend.publish(JSON.stringify(badLaoReq), id, channel)
+    And json answer = frontend.getBackendResponseWithoutBroadcast()
+    Then match answer contains INVALID_MESSAGE_FIELD
+    And match frontend.receiveNoMoreResponses() == true
```

Comment:
I wonder if the `== true` is necessary.
As well as the `id` argument.

Reply:
It does not seem to work without `==true` though